indico/modules/events/reminders/templates/reminders.html

```diff
@@ -43,9 +46,11 @@
             <div class="i-box-header">
                 <div class="i-box-title">{%- trans %}Pending Reminders{% endtrans -%}</div>
                 <div class="i-box-buttons">
-                    <a class="i-button icon-plus" href="{{ url_for('.add', event) }}">
-                        {%- trans %}Add Reminder{% endtrans -%}
-                    </a>
+                    <a class="i-button icon-plus" data-href="{{ url_for('.add', event) }}"
+                       data-title="{%- trans %}Add Reminder{% endtrans -%}"
+                       data-ajax-dialog
+                       data-reload-after>
+                       {%- trans %}Add Reminder{% endtrans -%}</a>
```

Comment:
not enough indentation (this is the tag's body, so it's 4 spaces), and the `</a>` goes into the next line